benchmarks/helics/EchoHubFederate.hpp

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright (c) 2017-2020,
+Battelle Memorial Institute; Lawrence Livermore National Security, LLC; Alliance for Sustainable Energy, LLC.  See
+the top-level NOTICE for additional details. All rights reserved.
+SPDX-License-Identifier: BSD-3-Clause
+*/
+
+#include "BenchmarkFederate.hpp"
+#include "helics/application_api/Inputs.hpp"
+#include "helics/application_api/Publications.hpp"
+#include "helics/application_api/Subscriptions.hpp"
+
+using helics::operator"" _t;
+/** class implementing the hub for an echo test*/
+class EchoHub: public BenchmarkFederate {
+  private:
+    std::vector<helics::Publication> pubs;
+    std::vector<helics::Input> subs;
+    int cnt = 10;
```

Comment:
this should be named something more descriptive and brace initialized with the default

Reply:
I renamed it to `num_leafs`